Snippets/ASB/ASB_7/DeadLetterQueueForwarding.cs

```diff
@@ -0,0 +1,15 @@
+using NServiceBus;
+
+class DeadLetterQueueForwarding
+{
+    void ConfigureDeadLetterQueueForwarding(EndpointConfiguration endpointConfiguration)
+    {
+        #region asb-configure-dead-letter-queue-forwarding
+
+        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
+        transport.Queues().ForwardDeadLetteredMessagesTo(queuename => queuename != "error" && queuename != "audit" && queuename != "centralizeddlq", "centralizeddlq");
+        transport.Subscriptions().ForwardDeadLetteredMessagesTo("centralizeddlq");
```

Comment:
Different rules for getting into DLQ. There's an ability to DLQ if no subscribers are found. That doesn't exist for queues.